axelor-sale/src/main/java/com/axelor/apps/sale/web/ConfiguratorCreatorController.java

```diff
@@ -69,4 +69,14 @@ public void importConfiguratorCreators(ActionRequest request, ActionResponse res
       TraceBackService.trace(e);
     }
   }
+
+  public void setFields(ActionRequest request, ActionResponse response) {
+
+    ConfiguratorCreator creator = request.getContext().asType(ConfiguratorCreator.class);
+    creator = Beans.get(ConfiguratorCreatorRepository.class).find(creator.getId());
+    ConfiguratorCreatorService configuratorCreatorService =
+        Beans.get(ConfiguratorCreatorService.class);
+    configuratorCreatorService.setFields(creator);
```

Comment:
you can just use `Beans.get(ConfiguratorCreatorService.class).setFields(creator);`